src/components/CurationPage/CurationPage.tsx

```diff
@@ -59,15 +59,21 @@ export default class CurationPage extends React.PureComponent<Props, State> {
   }
 
   renderAssigneeFilterDropdown = () => {
-    const { committeeMembers } = this.props
+    const { committeeMembers, wallet } = this.props
     const { assigneeFilter } = this.state
     return (
       <Dropdown
         direction="left"
         value={assigneeFilter}
         options={[
           { value: ALL_ASSIGNEES_KEY, text: t('curation_page.filter.all_assignees') },
-          ...committeeMembers.map(address => ({ value: address, text: <Profile textOnly address={address} /> }))
+          ...[
+            ...committeeMembers.filter(member => member === wallet.address),
```

Comment:
I'd use a reduce instead of 2 filters. Something like:
```
commMembers.reduce((acc, member) => {
  if (member === wallet.address) {
    acc.unshift(member)
  } else {
    acc.push(member)
  }
  return acc
},[])
```

Reply:
good idea! updated!